lib/dialects/better-sqlite3/index.js

```diff
@@ -7,6 +7,19 @@ class Client_BetterSQLite3 extends Client_SQLite3 {
     return require('better-sqlite3');
   }
 
+  /**
+   * Get `Version` from client or return `[0, 0, 0]` on failure.
+   *
+   * @returns {[number, number, number]}
+   */
+  _clientVersion() {
+    // hard coding 3.25.1 as better-sqlite3 offers no way to get Sqlite3 version it was built with
+    // and better-sqlite3 1.0.0 shipped with 3.25.1 in 2016.
+    //
+    // Issue opened for this: https://github.com/WiseLibs/better-sqlite3/issues/1122
+    return [3, 25, 1];
```

Comment:
can we require minimum better-sqlite with support for multi-insert via peerDependencies? I'd rather force people to upgrade rather than lock out them out of the beneficial feature

Reply:
So, two thoughts:

1. If `better-sqlite3` users are using the bundled `sqlite3` build then they're fine. The 1.0.0 version of `better-sqlite3` satisfies the 10+ year old requirement of being at least 3.11.7.
2. Technically the users of `better-sqlite3` (and `sqlite3` for that matter) can use a different C compilation of the sqlite3 library. So the NPM package version doesn't guarantee the version of `sqlite3.c` they've compiled.

What would you like to do?

Reply:
After https://github.com/WiseLibs/better-sqlite3/pull/1124 is merged we can do a more explicit check, but for older versions I would let user proceed by default and let them crash if their version is too old.